src/Core/Manifest/VersionProvider.php

```diff
@@ -201,34 +208,37 @@ public function getModuleVersionFromComposer($modules = [])
      * Load composer.lock's contents and return it
      *
      * @param bool $cache
-     * @return array
+     * @return array<mixed>
```

Comment:
```suggestion
     * @return array
```
Mixed is already implied so this doesn't add anything